tests/snuba/test_snuba.py

```diff
@@ -97,37 +97,6 @@ def test_organization_retention_larger_than_end_date(self):
                 },
             ) == {}
 
-    def test_use_group_id(self):
-        base_time = datetime.utcnow()
-        group = self.create_group()
-        self._insert_event_for_time(base_time, group_id=group.id)
-
-        with self.options({'snuba.use_group_id_column': True}):
```

Comment:
This `options` context manager flipped a switch that was on by default, and didn't do anything, so this didn't test anything other than we could issue queries.